schematic/models/validate_attribute.py

```diff
@@ -1406,12 +1504,18 @@ def _run_validation_across_targets_set(
         target_manifest: pd.core.series.Series,
         target_manifest_id: str,
         missing_manifest_log: dict[str, pd.core.series.Series],
-        present_manifest_log: dict[str, pd.core.series.Series],
+        present_manifest_log: list[str],
         repeat_manifest_log: dict[str, pd.core.series.Series],
+        target_attribute_in_manifest_list: list[bool],
+        target_manifest_empty: list[bool],
     ) -> tuple[
-        dict[str, pd.core.series.Series],
-        dict[str, pd.core.series.Series],
-        dict[str, pd.core.series.Series],
+        tuple[
+            dict[str, pd.core.series.Series],
+            dict[str, pd.core.series.Series],
```

Comment:
The second dict in this is tuple is the `present_manifest_log` which is a `list[str]`